src/js/core/languages.js

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
Why not read this from the werewolf-brain using `getLanguages()`? so we won't need an update when the brain gets new languages.